OpenRA.Game/StreamExts.cs

```diff
@@ -111,7 +154,12 @@ public static string ReadASCIIZ(this Stream s)
 			byte b;
 			while ((b = s.ReadUInt8()) != 0)
 				bytes.Add(b);
-			return new string(Encoding.ASCII.GetChars(bytes.ToArray()));
```

Comment:
is there a reason why this created a new string?

Reply:
I see it was like this from the very beggining https://github.com/OpenRA/OpenRA/commit/0c4df26ed0fa83f53cf56f374d36b6e89195ef9a #3414